main.js

```diff
@@ -220,18 +224,41 @@ document.querySelector('#hogCardContainer').addEventListener("click", (e)=> {
   const form= document.querySelector('form');
   form.addEventListener('submit',(e) => {
     e.preventDefault();
+    // const changeName = () => {
+    //   if("Gryffindor" !== "griff.jpg"){
+    //    return "Gryffindor"
+        
+    //   } 
+    //   if("Hufflepuff"!=="huff.jpg"){
+    //     return "Hufflepuff"
+    //   }
+    //   if("Ravenclaw"!=="raven.jpg"){
+    //     return "Ravenclaw"
+    //   }
+    //   if("Slytherin" !=="sly.jpg"){
+    //     return "Slytherin"
+    //   }
+    // }
     const houseNames= [
-      "Gryffindor", "Hufflepuff", "Ravenclaw","Slytherin"
+      "Gryffindor","Hufflepuff","Ravenclaw","Slytherin"
      ]
+
+     const houseImage= [ "griff.jpg","huff.jpg","raven.jpg","sly.jpg"
+     ]
+    //  "griff.jpg","huff.jpg","raven.jpg","sly.jpg"
+    //  changeName("griff.jpg"),changeName("huff.jpg"),changeName("raven.jpg"),changeName("sly.jpg")
     //  const getRandomInt = (min,max) =>{
     //   return Math.floor(Math.random() * (max-min)+ min)
     //  }
-     let sortHouse= houseNames [Math.floor(Math.random()* houseNames.length)]
+     let sortHouse= houseNames [Math.floor(Math.random()* houseNames.length)] && houseImage [Math.floor(Math.random()* houseImage.length)]
```

Comment:
This could result in a `houseName` not matching the `houseImage`. You need to first find the random index you want, then pass that same index to the `houseNames` and `houseImage` array.